### ISSUE TITLE
(2/19 - 18:00:00) expected 'SPENDER_DOES_NOT_HAVE_ALLOWANCE', got 'TOKEN_NOT_ASSOCIATED_TO_ACCOUNT' from round 162952161

### ISSUE THREAD

**Issue by iwsimon** [hashgraph/hedera-services#11849]
### Description

Transaction in HashScan [here](https://hashscan.io/mainnet/transaction/1708365932.290323003).

A crypto transfer transaction body:
```
transactionID {
  transactionValidStart {
    seconds: 1708365921
    nanos: 972917407
  }
  accountID {
    accountNum: 1061772
  }
}
nodeAccountID {
  accountNum: 18
}
transactionFee: 100000000
transactionValidDuration {
  seconds: 120
}
memo: "Altlantis - Live Mint"
cryptoTransfer {
  transfers {
    accountAmounts {
      accountID {
        accountNum: 1455021
      }
      amount: -10000
      is_approval: true
    }
    accountAmounts {
      accountID {
        accountNum: 1729620
      }
      amount: 10000
      is_approval: true
    }
  }
  tokenTransfers {
    token {
      tokenNum: 4604924
    }
    nftTransfers {
      senderAccountID {
        accountNum: 1729620
      }
      receiverAccountID {
        accountNum: 1455021
      }
      serialNumber: 55
      is_approval: true
    }
  }
}
```

Got

Mismatched values, expected 'SPENDER_DOES_NOT_HAVE_ALLOWANCE', got 'TOKEN_NOT_ASSOCIATED_TO_ACCOUNT' - Matching field 'status' Matching field 'receipt'  ==> expected: <SPENDER_DOES_NOT_HAVE_ALLOWANCE> but was: <TOKEN_NOT_ASSOCIATED_TO_ACCOUNT>

### Steps to reproduce

Run 15 mins event from round 162952161

### Hedera network

other

### Version

release/0.48

### Operating system

None

**Comment by iwsimon** [comment]
re-ran on 11852-pay-rewards-in-triggered-txn on 3/4/24 and got:
Mismatched values, expected 'AMOUNT_EXCEEDS_ALLOWANCE', got 'TOKEN_NOT_ASSOCIATED_TO_ACCOUNT' - Matching field 'status' Matching field 'receipt'  ==> expected: <AMOUNT_EXCEEDS_ALLOWANCE> but was: <TOKEN_NOT_ASSOCIATED_TO_ACCOUNT>. This should be fixed by pr #11848.

**Comment by iwsimon** [comment]
Re-tested today. This issue is fixed and open another issue for the same round.
